### DISCUSSION TITLE
how to save traces to mongodb in my spring app

### DISCUSSION THREAD

**Discussion by ayoubgoumbark** [open-telemetry/community#1500] [Community Discussions]
i want to save the traces id  of my  spring boot application in mongodb database using opentelemetry sdk

**Comment by trask** [comment]
hi @ayoubgoumbark! try asking in the OpenTelemetry Java community, either:

* [StackOverflow, tagged with `open-telemetry` and `java`](https://stackoverflow.com/questions/ask?tags=open-telemetry+java), or
* [CNCF Slack `#otel-java`](https://cloud-native.slack.com/archives/C014L2KCTE3)
